crates/ar-cli/src/collect_viewing_keys.rs

```diff
@@ -48,18 +46,16 @@ pub async fn run(
     let last_finalized_block_number = provider.get_block_number().await?;
     info!("last finalized block number: {last_finalized_block_number}");
 
-    let bytes = match endianess {
-        Endianess::LitteEndian => private_key_bytes(private_key_file)?,
-        Endianess::BigEndian => {
-            let mut bytes = private_key_bytes(private_key_file)?;
-            bytes.reverse();
-            bytes
-        }
+    let bytes = {
+        let mut bytes = private_key_bytes(private_key_file)?;
+        // We use big-endian encoding for the private key
```

Comment:
I need help here - why?

Reply:
This is for consistency with pub key, and pub key is for easier integration with foundry, because EVM uses big-endian. This all doesn't matter, but as long as it's consistent we are good.